src/main/java/budgetbuddy/storage/Storage.java

```diff
@@ -32,11 +34,16 @@
     Path getLoansFilePath();
 
     @Override
-    void saveLoans(LoansManager loansManager) throws IOException;
+    Path getAccountsFilePath();
 
     @Override
     Path getRuleFilePath();
 
     @Override
+    void saveLoans(LoansManager loansManager) throws IOException;
+
+    @Override
+    void saveAccounts(AccountsManager accountsManager) throws IOException;
+
     void saveRules(RuleManager ruleManager) throws IOException;
```

Comment:
I think you accidentally overwrote my `@Override` annotation when merging the code.

Reply:
I'll add it back